satpy/multiscene.py

```diff
@@ -46,54 +48,99 @@
 log = logging.getLogger(__name__)
 
 
-def stack(datasets, weights=None, combine_times=True):
-    """Overlay a series of datasets together.
+def stack(datasets, weights=None, combine_times=True, blend_type='select_with_weights'):
```

Comment:
I'm going through this code and trying to consolidate logic. I'm having trouble with this `combine_times` argument from @adybbroe's PR. @adybbroe any memory of why you/we had this `combine_times` keyword argument instead of just always doing it? I think @gerritholl had filed a bug report about regular stacking not combining the times already. I think I'll need to at least make the "no weights" case use this.